TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
>Anyways, I'd like to avoid dependencies where possible 
Totally agree with you! 

I guess kickstarting stuff is always tricky because we have to find a solution that works best for the majority of users out of the box.